jerry-core/parser/js/js-parser-statm.c

```diff
@@ -318,6 +326,30 @@ parser_parse_var_statement (parser_context_t *context_p) /**< context */
 
     if (context_p->token.type == LEXER_ASSIGN)
     {
+#ifdef JERRY_DEBUGGER
+      if (JERRY_CONTEXT (jerry_init_flags) & JERRY_INIT_DEBUGGER)
+      {
+        if (ident_line_counter != context_p->last_breakpoint_line)
+        {
+          JERRY_DEBUG_MSG ("Insert var breakpoint: %d (%d)\n", ident_line_counter, context_p->last_breakpoint_line);
```

Comment:
It will be printed to the stderr with the default port implementation. Check `jerry_port_log` implementation in `targets/default/jerry-port-default-io.c` for details.